### ISSUE TITLE
file .ck00MAN should not be in the tree

### ISSUE THREAD

**Issue by filbo** [lsof-org/lsof#93]
`.ck00MAN` is a build/install output file.  Its presence means that the 'manifest check' does not run unless you answer in the non-default affirmative to the second question in this dialog:
```
You can call the Inventory script directly at any time to take
inventory.  You can inhibit the inventory step permanently by
creating the file ./.neverInv, and you can tell the Configure script
to skip the inventory and customization steps with the -n option.

Do you want to take inventory (y|n) [y]? 

======================================================================

The lsof distribution inventory in 00MANIFEST has already been checked.

Do you want to check the inventory again (y|n) [n]? 
```
Running it generates the following error outputs (minus all the success outputs):
```
Examining .: ERROR
    File ./lsof.8 is missing. ++++
    File ./lsof.man is missing. ++++
Examining ./dialects/uw/uw7:
    Subdirectory ./dialects/uw/uw7/vm is missing. ++++
Examining ./dialects/uw/uw7/vm: ERROR
    Subdirectory ./dialects/uw/uw7/vm is missing. ++++
+++++++++++++++++++++++++++++++++++++++++++++++
+                                             +
+  SOME FILES OR DIRECTORIES MAY BE MISSING!  +
+                                             +
+++++++++++++++++++++++++++++++++++++++++++++++
```
Lack of running the manifest check has led to the manifest being out of date.  lsof.8 is now Lsof.8, lsof.man is no longer being provided, and the empty directory dialects/uw/uw7/vm no longer exists.  (These omissions affect the 'osr' and probably the 'osr6' and 'uw' `Configure` targets, but they aren't currently supported so that's OK.  I can step up to support those targets, if that's OK.)

To fix:

1. remove .ck00MAN

2. add empty directory dialects/uw/uw7/vm

3. figure out what to do about lsof.8 vs Lsof.8:

- 00MANIFEST mentions Lsof.8
- all of the individual dialect Makefiles look for lsof.8
- Lsof.8 is used only by support/makeman; the whole `support/` subdirectory appears to be Vic Abell's private lsof-FTP-site-maintenance suite

4. fix dialects/osr/Makefile to treat lsof.8 as other dialects do

I can submit changes later, just wanted to submit my findings before I lose track of them.

**Comment by masatake** [comment]
@filbo, are you interested in maintaining the uw dialect?

**Comment by filbo** [comment]
I can maintain the 'osr' (and 'osrgcc') dialect(s) with 10+ years experience working on that product; I can maintain 'osr6' and 'uw' with moderately passing familiarity + access to people with deep knowledge.

**Comment by masatake** [comment]
O.k. We should not delete osr and uw dialects as far as you are here.

**Comment by filbo** [comment]
I serious think you shouldn't delete any dialects.  You have a list of _supported_ dialects, which is shorter than the list of _implemented_ dialects.  If someone raises a bug about an unsupported dialect, the response is: ok, please fix it yourself (themself), or find a maintainer for it.  Not 'rip it out'.

**Comment by masatake** [comment]
I don't agree with you. However, I, the person who wants to remove unmaintained dialects, don't have enough time for removing the dialects. So I think the removing will not be occurred in lsof-4.x series.

**Comment by masatake** [comment]
I have invited you to this organization.

**Comment by dilinger** [comment]
I actually think we *should* delete the uw dialect in particular. See #274 for the rationale. In Debian, we've been manually stripping those headers out for years before packaging the lsof source.
